src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,347 @@ metadata_pkg.(
     is_wrapper = false
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+`LinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features. Linear coefficients
+are calculated using least squares.
+Options exist to specify a bias term.
```

Comment:
I think that as this is just linear least squares, there is not need to formulate an assumption about how data is generated. Yes, under that assumption least squares corresponds to maximum likelihood estimate, and so forth. But the model description is already unambiguous by saying "least squares". And this model does not report any of the extra stuff you would normally expect under the standard data generation model (eg, covariance matrix). 

So I suggest:

```suggestion
`LinearRegressor` assumes the target is a continuous variable
whose conditional distribution is normal with constant variance, and whose
expected value is a linear combination of the features. Linear coefficients
are calculated using least squares.
Options exist to specify a bias term.
```

Reply:
Sorry @josephsdavid I forgot to edit the suggestion 😳. Edited the suggestion to fix